### PR TITLE
Adding CI/CD pipiline #7

### PR DESCRIPTION
**Pull request by NidhiHemanth**
Adding CI/CD pipilines aka github Actions.

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build & Run
+      run: cargo run --bin snapshot --release
```
Comment (sn99) [inline .github/workflows/rust.yml:20]: Just `cargo build --release` is enough

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # windows-snapshot
 
+[![Rust](https://github.com/NidhiHemanth/windows-snapshot/actions/workflows/rust.yml/badge.svg)](https://github.com/NidhiHemanth/windows-snapshot/actions/workflows/rust.yml)
```
Comment (sn99) [inline README.md:3]: The link and all might have to be changed for SubCom later, rn it shows status of your fork not the master

**Review by sn99 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (NidhiHemanth) [reply README.md:3]: Will make the changes after the PR gets merged

Comment (sn99) [reply README.md:3]: Smart

**Pull request merged**